app/models/curry/unauthorized_commit_author_comment.rb

```diff
@@ -44,7 +44,7 @@ def comment_body
         verified email address. To become authorized to contribute, you will
         need to sign the Contributor License Agreement (CLA) as an individual or
         on behalf of your company. [You can read more on Chef's
-        blog.](#{chef_blog_url('2014/06/23/changes-to-the-contributor-license-agreement-process')})
+        blog.](https://www.chef.io/blog/2014/06/23/changes-to-the-contributor-license-agreement-process)
```

Comment:
Would it be better to `include CustomUrlHelper` in this class? I'm very :-1: on hard-coding URLs.